core/frontend/src/components/parameter-editor/ParameterEditor.vue

```diff
@@ -346,6 +353,53 @@ export default Vue.extend({
         return 'N/A'
       }
     },
+    saveParametersToFile() {
+      let parameter_name_max_size = 0
+
+      // Sort parameters alphabetically
+      // We take advantage of sort to also calculate the parameter name maximum size
+      const parameters = Object.values(autopilot_data.parameters).sort((first, second) => {
+        if (parameter_name_max_size < first.name.length) {
+          parameter_name_max_size = first.name.length
+        }
+        if (parameter_name_max_size < second.name.length) {
+          parameter_name_max_size = second.name.length
+        }
+
```

Comment:
```suggestion
This is functionally the same thing, right?
```suggestion
        parameter_name_max_size = Math.max([parameter_name_max_size, first.name.length, second.name.length])
```

```

Reply:
Nice!